terraform/deployment/main.tf

```diff
@@ -346,6 +346,71 @@ module "ecs_nestjs_app" {
   }
 }
 
+module "ecs_actix_app" {
+  source       = "./modules/ecs"
+  alb_listener = module.public_alb.alb_listener
+  alb = {
+    target_group       = "actix-tg"
+    target_group_paths = ["/actix/*"]
+    arn                = module.public_alb.alb_listener_http_tcp_arn
+    rule_priority      = 1
```

Comment:
Rule priority cannot be 1. It has to be +1 of the highest number we already have

Reply:
Fixed ... also found the issue for the nest-js part and adapted it.